gitgud/levels/intro/_rebasing/instructions.txt

```diff
@@ -0,0 +1,11 @@
+This level will familiarize you with the concept of git rebasing!
+>>>
+Rebasing in git is an alternative way of combining two different branches (recall the last method: merging).
+>>>
+In the current tree, you can see that our commit tree has two different branches, "master" and "bugFix", both diverging from node 1.
+>>>
+In order to merge bugFix into master, we checkout the bugFix branch, and run "git rebase master". This will copy the bugFix branch below master, using master as it's parent node. The command "git log --graph --oneline" is a useful tool to observe the new "flattened" commit tree.
```

Comment:
The good things first, then I'll nitpick some of the details with rebasing. 
I like how you say it's an alternative way of combining two different branches, that's exactly what it is
I like your inclusion of "git log --graph --oneline". This is a really useful command for visualizing everything and will be especially important later on, but keep in mind, the --graph option won't be particularly useful here since, in this level there are no merge commits. The graph will only ever split up if there's a merge commit. Since that's not already in the merging challenge, I think it'd be a great idea to add that there too and then we can highlight the difference between merging and rebasing. If you want, you should go add that command to the instructions for merging and add that change to your branch so it can be included in this PR. That would be great. Thanks.

Now for the nitpicking:
I think in the instructions it's important to make sure we're getting the explanation right, so I think it's best if you remove this:
"In order to merge bugFix into master, we checkout the bugFix branch, and run "git rebase master". This will copy the bugFix branch below master, using master as it's parent node."

To be clear, rebasing is not a merge and it's not used in exactly the same situations. You can rebase instead of a merge, but a rebase isn't a merge and we shouldn't call it one. Below, I've written how I would describe the rebase we're performing. You can use that to help write some of the instructions

Sometimes, when we're  writing code, there's new code that we want to incorporate into our own, but we don't always want to use a merge. For this situation, there is rebasing. Rebasing "bases" your code off of a different commit. When we started working on bugFix, we were only on commit 1. And our branch is "based" off of commit 1. Now someone (perhaps us) has added two commits to the master branch and we want them, so we rebase bugFix off of that new work, so now, when we look at the history, we can see all of the commits from the master branch and all of our commits from the bugFix branch

Reply:
Sorry if this is a bit complicated. I appreciate your hard work! I hope you're learning something too!